newscout_web/api/v1/views.py

```diff
@@ -1020,6 +1020,15 @@ def publish(self, obj):
 
     def post(self, request):
         publish = request.data.get("publish")
+        origin = request.META.get("HTTP_ORIGIN")
```

Comment:
Just comment the usage of `origin` in the code

Reply:
fixed in commit id: b336df54a91d0a1f30b64f8b703039aff6191f5d